### DISCUSSION TITLE
Image Resizing

### DISCUSSION THREAD

**Discussion by giliomeejg** [gitcoinco/grants-stack#1129] [Ideas]
The previous Alpha Round saw some pages loading with 100Mb of un-resized images.  
This is detrimental to under served communities that fork out a fair bit of cash for bandwidth.

It appears that Pinata, which is already used, has an image resizing service that should be fairly easy to implement by simply adding '?img-width=200&img-height=100' to the end of the URLs being requested:
https://medium.com/pinata/resizing-ipfs-images-with-pinatas-image-optimization-tools-fb381bee58aa

As an example, here's an image that was loaded during the round, and resized on the client side to 296x120:
[https://gitcoin.mypinata.cloud/ipfs/bafy...vchr2maura](https://gitcoin.mypinata.cloud/ipfs/bafybeihnzpm7yykjwl66ko44wmmortqe45radvjzuil7g2fwvchr2maura) (3.9Mb)

Here's the same image, resized appropriately:
[https://gitcoin.mypinata.cloud/ipfs/bafy...vchr2maura?img-width=296&img-height=120](https://gitcoin.mypinata.cloud/ipfs/bafybeihnzpm7yykjwl66ko44wmmortqe45radvjzuil7g2fwvchr2maura?img-width=296&img-height=120) (21Kb)

This is a 99.4% reduction in bandwidth and a relatively simple implementation.

**Comment by melissa-neira** [comment]
Totally on board with this @giliomeejg! cc: @thelostone-mc @nfrgosselin

**Comment by vacekj** [comment]
Thank you for bringing the Pinata service to our attentiont, we will definitely use this.

**Comment by vacekj** [comment]
Also investigate client-side image resizing before upload, as this will significantly reduce our costs during upload phase.

**Comment by vacekj** [comment]
Closing this: we are now using pinata resizing + AWS CloudFront, and I believe this combination serves our needs perfectly.
